Cargo.toml

```diff
@@ -4,11 +4,11 @@ version = "1.115.0"
 publish = false
 
 [[bin]]
-name = "fxa-email-service"
+name = "fxa_email_send"
```

Comment:
I had to call this something different to prevent a name clash in the generated docs between this bin and the main lib containing all the types etc.